### DISCUSSION TITLE
Downloading, how it works exactly?

### DISCUSSION THREAD

**Discussion by linuxdep** [Shabinder/SpotiFlyer#595] [General]
How exactly do the downloads take place? If they are done in parallel, there is a risk that the account will be blocked, as has already happened with other software. Or does SpotiFlyer work differently?
It would be a good idea to simply cut the stream 1:1. Then it would look like someone was listening to it.

**Comment by Shabinder** [comment]
which account do you expect to get blocked ?
there is **NO sign in** taking place.

So for every song downloading starts, it happens from an **anonymous account.**

In some cases when people download a massive amount of songs like 1k altogether, YT might start sending `429 `, if that happens just wait a few hours and you are good to go

In no case there is permanent ban of any kind.

**Reply by askfriends** [reply]
> which account do you expect to get blocked ? there is **NO sign in** taking place.
> 
> So for every song downloading starts, it happens from an **anonymous account.**
> 
> In some cases when people download a massive amount of songs like 1k altogether, YT might start sending `429 `, if that happens just wait a few hours and you are good to go
> 
> In no case there is permanent ban of any kind.

so basically this app downloads songs from YT Music and not from spotify,gaana etc servers?

**Reply by Shabinder** [reply]
Yes for now we get files from JioSaavn, YT-Music and SoundCloud

**Reply by askfriends** [reply]
> > which account do you expect to get blocked ? there is **NO sign in** taking place.
> > So for every song downloading starts, it happens from an **anonymous account.**
> > In some cases when people download a massive amount of songs like 1k altogether, YT might start sending `429 `, if that happens just wait a few hours and you are good to go
> > In no case there is permanent ban of any kind.
> 
> so basically this app downloads songs from YT Music and not from spotify,gaana etc servers?

I see..
BTW is ther any way to download from original servers?

**Comment by sapreen555** [comment]
great

https://hl-tourism.com/
